### PR TITLE
[nvidia-bluefield] Add dependencies required for Nvidia-bluefield image compilation.

### PR DESCRIPTION
**Pull request by oleksandrivantsiv**
#### Why I did it
Add dependencies required for Nvidia-bluefield image compilation.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Extend a list of the DEB packages installed in the Bookworm slave docker container.

#### How to verify it
Compile Bookworm slave docker container.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

**Comment by oleksandrivantsiv** [comment]
Related to https://github.com/sonic-net/sonic-buildimage/pull/18466

File: sonic-slave-bookworm/Dockerfile.j2 [hunk 1]
```diff
@@ -420,7 +420,30 @@ RUN apt-get update && apt-get install -y \
...
+{%- endif %}
+        efibootmgr \
+        watchdog \
+        dmidecode \
+# For nvidia-blufield sdk compilation
+        cython3 \
+        pandoc \
+        valgrind \
+        udev \
+        inkscape \
```
Comment (saiarcot895) [inline sonic-slave-bookworm/Dockerfile.j2:441]: Why does the SDK require `valgrind` and `inkscape` for compilation?

File: sonic-slave-bookworm/Dockerfile.j2 [hunk 2]
```diff
@@ -420,7 +420,30 @@ RUN apt-get update && apt-get install -y \
...
+        meson \
+        python3-pyelftools \
+        hwdata \
+        libzip-dev \
+        libzip4
```
Comment (saiarcot895) [inline sonic-slave-bookworm/Dockerfile.j2:446]: Does `lipzip4` need to be listed out separately here? `libzip-dev` should bring in the library automatically.

File: sonic-slave-bookworm/Dockerfile.j2 (hunk 1 above)
Comment (oleksandrivantsiv) [reply sonic-slave-bookworm/Dockerfile.j2:441]: `valgrind` is required for OFED compilation
`inkscape` is required for DPDK compilation

Comment (oleksandrivantsiv) [reply sonic-slave-bookworm/Dockerfile.j2:441]: DPDK uses `inkscape` for the memory reports.

Comment (oleksandrivantsiv) [reply sonic-slave-bookworm/Dockerfile.j2:441]: OFED uses `valgrind` to provide built-in debugging capabilities.

File: sonic-slave-bookworm/Dockerfile.j2 (hunk 2 above)
Comment (oleksandrivantsiv) [reply sonic-slave-bookworm/Dockerfile.j2]: You are right. I removed `lipzip4`

**Comment by dgsudharsan** [comment]
Please restart the pipeline. Mellanox build should pass now

**Comment by liat-grozovik** [comment]
@oleksandrivantsiv is there any dependency to avoid the checker failures? if not, can you please review them?

**Review by liat-grozovik — APPROVED** [review]

**Comment by liat-grozovik** [comment]
@saiarcot895 can you approve?

**Review by saiarcot895 — APPROVED** [review]

**Pull request merged**